### DISCUSSION TITLE
Edit own userprofile (/users/editOwn/) ends up with error 500

### DISCUSSION THREAD

**Discussion by dlx295** [Leantime/leantime#1904] [Q&A]
Hi,

when I try to edit my own details at the /users/editOwn/ page, I am getten error 500 from webserver. 

Everything else seems to work. 

I am running leantime in a docker environment on my synology nas.

`
127.0.0.1 -  25/Sep/2023:05:12:33 +0000 "GET /index.php" 200
`

**Comment by marcelfolaron** [comment]
Doing some house keeping. This was resolved
